server/app/views/applicant/ProgramIndexView.java

```diff
@@ -228,7 +234,8 @@ private DivTag programCardsSection(
       long applicantId,
       Locale preferredLocale,
       ImmutableList<ApplicantService.ApplicantProgramData> cards,
-      MessageKey applyTitle) {
+      MessageKey applyTitle,
```

Comment:
nit: should we rename these to "buttonTitle" "buttonSrText"?

Reply:
Done.